be/src/exec/pipeline/aggregate/aggregate_blocking_source_operator.h

```diff
@@ -50,7 +50,11 @@ class AggregateBlockingSourceOperatorFactory final : public SourceOperatorFactor
                                                                  _aggregator_factory->get_or_create(driver_sequence));
     }
 
+    bool need_local_shuffle() const override { return _need_local_shuffle; }
+    void set_need_local_shuffle(bool need_local_shuffle) { _need_local_shuffle = need_local_shuffle; }
+
 private:
     AggregatorFactoryPtr _aggregator_factory = nullptr;
+    bool _need_local_shuffle = true;
```

Comment:
Where this flag comes from?

Reply:
It comes from the source operator of `AggregateBlockingSinkOperator`.

For example, as for `(SourceA⇨AggregateBlockingSinkOperator)  AggregateBlockingSourceOperator⇨Other`, `need_local_shuffle` is inherited from `SourceA`.

I would add some comments.